t/Test30-make_coupler_mosaic_stretched_grid.sh

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env bats
+# test land mask stretched grid check area ratio 
+
+@test "Test land mask stretched grid data lat 32.0 34.0 35.4" {
+
+  if [ ! -d "Test30" ] 
+  then
+  		mkdir Test30
+  fi
+
+  cd Test30
+  
+  cp $top_srcdir/t/Test30-input/ocean_hgrid.nc . 
+  cp $top_srcdir/t/Test30-input/ocean_mosaic.nc .
+  cp $top_srcdir/t/Test30-input/ocean_topog.nc .
+
+#Make no streetched grid 
+  run command make_hgrid --grid_type gnomonic_ed \
+                         --do_schmidt --stretch_factor 2.5 \
+                         --target_lon 262.4 \
+                         --target_lat 32.0 \
+                         --nlon 512 \
+                         --grid_name C256_grid_32.0
+  [ "$status" -eq 0 ]
+
+  run command make_hgrid --grid_type gnomonic_ed \
+                         --do_schmidt --stretch_factor 2.5 \
+                         --target_lon 262.4 \
+                         --target_lat 34.0 \
+                         --nlon 512 \
+                         --grid_name C256_grid_34.0
+  [ "$status" -eq 0 ]
+
+  run command make_hgrid --grid_type gnomonic_ed \
+                         --do_schmidt --stretch_factor 2.5 \
+                         --target_lon 262.4 \
+                         --target_lat 35.4 \
+                         --nlon 512 \
+                         --grid_name C256_grid_35.4
+  [ "$status" -eq 0 ]
+
+#Create no stretched grid mosaic
```

Comment:
Perhaps change the test30 filename to not include "stretched" as the test includes regular non-stretched grids as well

Reply:
removed "stretched" from the file name.